src/app/shared/services/switchboard-toaster.service.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { TestBed } from '@angular/core/testing';
+
+import { SwitchboardToasterService } from './switchboard-toaster.service';
+import { ToastrModule, ToastrService } from 'ngx-toastr';
+
+describe('SwitchboardToasterService', () => {
+  let service: SwitchboardToasterService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [ToastrService],
```

Comment:
I think you should provide a Toastr mock, not real service. I think we should test our services, not libraries.